src/QMCWaveFunctions/tests/test_MO_spinor.cpp

```diff
@@ -271,8 +275,11 @@ void test_lcao_spinor()
   for (int iat = 0; iat < 1; iat++)
   {
     std::vector<ParticleSet::SingleParticlePos> displs = {dR[iat], dR[iat]};
+    std::vector<ParticleSet::Scalar_t> sdispls = {dS[iat], dS[iat]};
     elec_.mw_makeMove(p_list, iat, displs);
+    elec_.mw_makeSpinMove(p_list, iat, sdispls);
```

Comment:
Could update this test to call the new APIs.

Reply:
yea, I was just working on this

Reply:
fixed